cpp/ql/test/query-tests/Security/CWE/CWE-190/semmle/TaintedAllocationSize/TaintedAllocationSize.expected

```diff
@@ -143,8 +141,8 @@ nodes
 | test.cpp:231:9:231:24 | call to get_tainted_size | semmle.label | call to get_tainted_size |
 | test.cpp:231:9:231:24 | call to get_tainted_size | semmle.label | call to get_tainted_size |
 | test.cpp:231:9:231:24 | call to get_tainted_size | semmle.label | call to get_tainted_size |
-| test.cpp:235:11:235:20 | (size_t)... | semmle.label | (size_t)... |
-| test.cpp:237:10:237:19 | (size_t)... | semmle.label | (size_t)... |
+| test.cpp:235:2:235:9 | Arg(0) | semmle.label | Arg(0) |
+| test.cpp:237:2:237:8 | Arg(0) | semmle.label | Arg(0) |
```

Comment:
These `toString`s were not very pretty before, and they're not very pretty now. But at least now we have the ability to override their `toString` in the future because we've separated the argument from the instruction that produces it.

Reply:
I just noticed this as well and pushed a better `toString`. Originally, I had planned on adding this in a future PR, but I couldn't accept these ugly path explanations we got without a specialized `toString`.